Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -613,7 +624,7 @@ private void RaycastUnityUI(PointerData pointer, LayerMask[] prioritizedLayerMas
             }
 
             // Check if we need to overwrite the physics raycast info
-            if ((pointer.End.Object == null || overridePhysicsRaycast) && uiRaycastResult.module.eventCamera == UIRaycastCamera)
+            if ((pointer.End.Object == null || overridePhysicsRaycast) && (uiRaycastResult.module != null && uiRaycastResult.module.eventCamera == UIRaycastCamera))
```

Comment:
This is also fixed in 1350

Reply:
Exactly, we want it to be undisruptive to other pointers.

>This is also fixed in 1350

Got it, I'll rebase once 1350 is merged.

Reply:
`uiRaycastResult.IsValid`